### PR TITLE
awesome.quit(): Add exit code argument

### PR DESCRIPTION
**Pull request by psychon**
Fixes: https://github.com/awesomeWM/awesome/issues/1184
Signed-off-by: Uli Schlachter <psychon@znc.in>

### CODE REVIEW

File: luaa.c [hunk 1]
```diff
@@ -155,11 +155,14 @@ composite_manager_running(void)
 }
 
 /** Quit awesome.
+ * @tparam[opt=0] integer code The exit to use when exiting.
```
Comment (blueyed) [inline luaa.c]: s/exit/exit code/
Or just "code".

Comment (psychon) [inline luaa.c]: Whoops...

**Comment by psychon** [comment]
Squashed:
```diff
diff --git a/luaa.c b/luaa.c
index 1763f40..e5cd055 100644
--- a/luaa.c
+++ b/luaa.c
@@ -155,7 +155,7 @@ composite_manager_running(void)
 }
 
 /** Quit awesome.
- * @tparam[opt=0] integer code The exit to use when exiting.
+ * @tparam[opt=0] integer code The exit code to use when exiting.
  * @function quit
  */
 static int
```

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**